flight/src/types.rs

```diff
@@ -0,0 +1,39 @@
+use cid::Cid;
+use serde::{Deserialize, Serialize};
+use serde_bytes::ByteBuf;
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub enum ConclusionEvent {
+    Data(ConclusionData),
+    Time(TimeEvent),
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct ConclusionInit {
+    pub stream_type: String,
+    pub controllers: String,
+    pub dimensions: Vec<(String, ByteBuf)>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct ConclusionData {
+    pub id: Cid,
+    pub init: ConclusionInit,
+    // TODO : rethink this, add a checkpoint to make it work in datafusion query
+    pub previous: Vec<Cid>,
+    pub before: Option<CeramicTime>,
+    pub after: Option<CeramicTime>,
```

Comment:
Lets remove both `before` and `after` from the data events. We have decided that we are not going to support conclusions about the temporality of data events. Consumers of the feed can make their own conclusions when a time event is emitted.